spec/acceptance/enum_traits_spec.rb

```diff
@@ -0,0 +1,161 @@
+describe "enum traits" do
+  context "when automatically_define_enum_traits is true" do
+    it "builds traits automatically for model enum field" do
+      define_model("Task", status: :integer) do
+        enum status: { queued: 0, started: 1, finished: 2 }
```

Comment:
these are somewhat also redundant, so they are used in every `it` the same or just the keys as a string array, it may make it more readable to do a single `let` for them for the whole `describe` block (again obviously not high prio)

Reply:
I am generally not a fan of `let` for the reasons outlined in this post: https://thoughtbot.com/blog/lets-not.